### ISSUE TITLE
GPUImage module not found

### ISSUE THREAD

**Issue by fraserqed** [garnele007/SwiftOCR#39]
![image](https://cloud.githubusercontent.com/assets/20955337/17565770/c6f4eeee-5f06-11e6-8fdf-681e5a7e6458.png)

I'm found similar issues around but none of the solutions seem to work. I get the error "No such module 'GPUImage'" when building the final product on all examples. I can build all the frameworks with no problem. Also, GPUImage framework is included in the embedded binaries and and linked frameworks. I'm using XCode Version 7.3.1 (7D1014), OS X 10.11.3 on a MacBook Air.

Thanks!

**Comment by garnele007** [comment]
Could not reproduce. Did you try to clean your build folder?

**Comment by fraserqed** [comment]
Yes, cleaned successfully. Also, a colleague can run the same zipped project folder on a different machine with no problems. My GPU is an Intel HD5000 but I would hope that wouldn't matter.

**Comment by garnele007** [comment]
Hmmm... I have no idea why it doesn't work but I found this: http://stackoverflow.com/questions/29500227/xcode-no-such-module-error-but-the-framework-is-there

You could also try to delete your Derived Data directory (`~/Library/Developer/Xcode/DerivedData`) and/or adding `$'inherited'` to Framework Search Path.

**Comment by KevinGong2013** [comment]
@fraserqed  Do not download zip directly,  use `git clone` and `git submodule update --init`

**Comment by Lucien** [comment]
This happened to me when I had another project running and also imported GPUImage. Closing the other project and running SwiftOCR again Training fixed.

**Comment by jbree** [comment]
I was having this issue tonight building with Carthage for a Mac app. Revoking my old cert and getting a new one cleared it up.

**Comment by imaddi47** [comment]
Have you solved this issue
